.circleci/config.yml

```diff
@@ -422,6 +417,32 @@ jobs:
           background: true
           paths:
             - ./habitat-api
+      - run:
+          name: Build api documentation
+          command: |
+              export PATH=$HOME/miniconda/bin:$PATH
+              
+              . activate habitat; cd habitat-api
+              python setup.py develop --all
+
+              # NOTE: Temporary workaround for above line while https://github.com/facebookresearch/habitat-api/issues/385 is unresolved
+              # Create conda env with python 3.8 to build habitat-api documentation 
+              conda create -y -n habitat-py38 python=3.8
+              . activate habitat-py38; #cd habitat-api
+              conda install -q -y -c conda-forge ninja numpy pytest pytest-cov ccache hypothesis
+              pip install -r requirements.txt --progress-bar off
```

Comment:
Shouldn't be needed anymore as facebookresearch/habitat-api#385 is fixed by #627.

Reply:
Addressed.